### PR TITLE
Random Fixes

### PR DESCRIPTION
**Pull request by brunobottazzini**
Contains Fixes fro:
- Running FBP using environment file
- Change variables name when creating file
- Disabble save button when the file is not possible to be saved

### CODE REVIEW

File: server/views/index.html [hunk 1]
```diff
@@ -162,7 +162,7 @@
                                  </md-menu-item>
                                  <md-menu-divider></md-menu-divider>
                                  <md-menu-item>
-                                    <md-button ng-click="ctrl.menuAction('file.save', $event)">
+                                    <md-button ng-disabled="!shouldSave" ng-click="ctrl.menuAction('file.save', $event)">
```
Comment (gpaes) [inline server/views/index.html:165]: When can we reach the situation that the file can't be saved?

Comment (brunobottazzini) [inline server/views/index.html:165]: Situations where the files can't be saved are:
   1 - The FBP file is inside the demo folder.
   2 - The FBP does not belong to any project and the file wasn't created before. (This happens when you start coding without importing or creating a new project).

Some improvements for the number 2 were suggested here: https://github.com/solettaproject/soletta-dev-app/issues/85

**Comment by gpaes** [comment]
@brunobottazzini Ok, #85 solves the second point.
For the first one, what does that mean? The person can't modify a demo? Or the person can't save a file created inside the demo folder? If the person managed to create the file why can't she save it?

**Comment by brunobottazzini** [comment]
1- Demo can be changed but not saved.
2- What do you mean with "can't save?" If the user created a file in an imported or new project he will be able to save it.

**Pull request merged**